tests/MoBi.Tests/Core/BaseForSpecs.cs

```diff
@@ -1,177 +0,0 @@
-﻿using System.Xml.Linq;
-using MoBi.Core.Domain.Model;
-using NUnit.Framework;
-using OSPSuite.Core.Domain;
-using OSPSuite.Core.Domain.Formulas;
-using OSPSuite.Core.Domain.UnitSystem;
-using OSPSuite.Core.Serialization;
-using OSPSuite.Core.Serialization.Xml;
-using OSPSuite.Serializer.Xml;
-using OSPSuite.Utility.Container;
-using IContainer = OSPSuite.Core.Domain.IContainer;
-
-namespace MoBi.Core
-{
-   public class SpecsBaseWithModel<T_Serializer, T_Object> : SpecsBase<T_Serializer, T_Object>
```

Comment:
No idea what this is all about. This was not used and clearly does not match what we do normally so kill